ansible_collections/arista/avd/plugins/README.md

```diff
@@ -73,24 +73,25 @@ To use this filter:
 
 ### convert_dicts filter
 
-The `arista.avd.convert_dicts` filter will convert a dictionary containing nested dictionaries to a list of dictionaries
-It inserts the outer dictionary keys into each list item using the primary_key `name` (key name is configurable) and if there is a list of dictionary in the dictionary value, it inserts this value list to secondary key "items" (key name is configurable).
+The `arista.avd.convert_dicts` filter will convert a dictionary containing nested dictionaries to a list of
+dictionaries.It inserts the outer dictionary keys into each list item using the primary_key `name` (key name is
```

Comment:
missing a space between `dictionnaries.` and `It`

Reply:
fixed.